internal/router/controllers/controller.go

```diff
@@ -30,15 +29,13 @@ type SQLRunner interface {
 
 // Controller defines the HTTP handlers for interacting with user tables.
 type Controller struct {
```

Comment:
The controller had these two dependencies: `SQLRunner` (implemented by the old `mesa.go`) and `system.SystemService`. It's all `Gateway` now.